default.nix

```diff
@@ -374,7 +374,32 @@ in rec {
       linuxExe = serverOn (projectOut "x86_64-linux");
       dummyVersion = "Version number is only available for deployments";
     in mainProjectOut // {
-      profiled = projectOut { inherit system; enableLibraryProfiling = true; };
+
+      profiledObRun = let
```

Comment:
I'd feel better if this were in the `internal` attrset or similar since I don't really want to provide any guarantees about this API.

Reply:
We need to make sure this is added to the cache outputs in `release.nix`.

Reply:
It is added to all-builds.nix for skeleton at least